packages/context/src/context.ts

```diff
@@ -66,6 +66,153 @@ export class Context {
     return binding;
   }
 
+  /**
+   * Create a corresponding binding for configuration of the target bound by
+   * the given key in the context.
+   *
+   * For example, `ctx.configure('controllers.MyController', {x: 1})` will
+   * create binding `controllers.MyController:$config` with value `{x: 1}`.
+   *
+   * @param key The key for the binding that accepts the config
+   * @param config Options object
+   */
+  configure(key: string, config?: BoundValue): Binding {
+    const keyForOptions = Binding.buildKeyForConfig(key);
+    const bindingForOptions = this.bind(keyForOptions);
+    if (config != null) {
+      bindingForOptions.to(config).tag(`config:${key}`);
```

Comment:
I think the tag should be applied always, even when `config` value was not provided (e.g. because the value is configured via `.toClass` etc.).

Reply:
Fixed.